mungegithub/mungegithub.go

```diff
@@ -107,6 +107,8 @@ func main() {
 			if err := mungers.InitializeMungers(&config.Config, &config.Features); err != nil {
 				glog.Fatalf("unable to initialize mungers: %v", err)
 			}
+			config.HookHandler.Initialize()
+			go config.HookHandler.Listen()
```

Comment:
Didn't you have code to make it all work without turning the web hook on? Do we need such code? Is there a reason not to put the `Listen()` inside the `Initialize()`?